resources/css/utilities.css

```diff
@@ -52,3 +52,15 @@ span.smalldate {
 [x-cloak] {
   display: none !important;
 }
+
+.dot-low {
+  color: var(--dot-color-low);
+}
+
+.dot-warning {
+  color: var(--dot-color-warning);
+}
+
+.dot-urgent {
+  color: var(--dot-color-urgent);
+}
```

Comment:
Using global CSS for this is probably overkill. Tailwind utilities alone are enough to get by.

However, I don't believe this is desirable. Ascribing severity to the dot based solely on count of the items may miss the mark. There may be only 1 item, but it's a critical issue. We also really shouldn't convey such information using color- it's an accessibility violation.

For the time being, I suggest we keep things simple and have the dot represent the presence of notifications.

Reply:
The colour is based on the `max()` value of the new priority key that is found in the `$notifications` collection, not the count of notifications found. So if you had 12 messages it would be blue, add one Ticket awaiting feedback and the dot would be Orange and if you then got a message with priority 2 (say a message from an admin account) then the dot would go red.

That said your right about it being an accessability issue so I'll go ahead and roll that back.